reference/content/driver-reactive/index.md

```diff
@@ -26,6 +26,7 @@ For tutorials for some frequently used operations, see the [Tutorials section]({
 
 | Release | Documentation |
 |---------|---------------|
+| 4.5.x   | {{< releases version="4.5" >}} |
```

Comment:
should 4.6 be on here?

Reply:
This is a list of past version links. They allow you to switch context of the site to an older version.